testing/test_mud.sh

```diff
@@ -47,17 +47,43 @@ function test_device_traffic {
     echo device-$device_num $type $((bfr_peer > 2)) $((bfr_ngbr > 0)) $((ufr_peer > 2)) $((ufr_ngbr > 0)) | tee -a $TEST_RESULTS
 }
 
+function test_acl_count {
+    device_num=$1
+    peer_num=$((3-device_num))
+    device_mac=9a:02:57:1e:8f:0$device_num
+    peer_mac=9a:02:57:1e:8f:0$peer_num
+    metric_output_file=inst/run-9a02571e8f0$device_num/metric_output.json
+    rule_filter=eth_src=$peer_mac,eth_dst=$device_mac
+
+    python3 daq/varz_state_collector.py -y flow_packet_count_port_acl -o $metric_output_file -l $rule_filter
+    packet_count=$(jq '.gauge_metrics.flow_packet_count_port_acl.samples[0].value' $metric_output_file)
+    echo device-$device_num $type $((packet_count > 2)) | tee -a $TEST_RESULTS
+}
+
+function terminate_daq {
+    daq_pid=$(<inst/daq.pid)
+    sudo kill -SIGINT $daq_pid
+    while [ -f inst/daq.pid ]; do
+        sleep 5
```

Comment:
add some logging message here in the loop so if it gets stuck for some reason it's more obvious in the logs...

Reply:
Done. PTAL..